classes/class-nwcc-loader.php

```diff
@@ -0,0 +1,358 @@
+<?php
+/**
+ * NWCC Loader.
+ *
+ * @package NWCC
+ */
+if( ! class_exists( 'Multisite_Script_Class' ) ) {
+
+	/**
+	 * Multisite_Script_Class initial setup
+	 */
+	class Multisite_Script_Class{
+
+		/**
+		 * Member Variable for update sitewide option
+		 *
+		 * @var multisite_script_option
+		 */
+		private $multisite_script_option;
+
+		/**
+		 * Member Variable for getting current site ID
+		 *
+		 * @var current_blog
+		 */
+		private $current_blog;
+		
+		/**
+		 *  Constructor
+		 */
+		public function __construct() {
+
+            $this->current_blog = get_current_blog_id();
+            switch_to_blog( 1 );
+            $this->multisite_script_option = get_site_option( 'multisite_script_option' );
+            switch_to_blog( $this->current_blog );
+
+            // Add required actions.
+            add_action( 'network_admin_menu', array( $this, 'add_settings_page' ), 9999 );
+            add_action( 'admin_init', array( $this, 'register_settings' ) );
+            add_action( 'init', array( $this, 'admin_post_edit_options' ) );
+            add_action( 'init', array( $this, 'run_loader' ) );
+
+            // Load textdomain translations.
+            $this->load_textdomain();
```

Comment:
Need to correct indentation for this block of code.

Reply:
Code is correctly intended here - https://github.com/imnavanath/network-wide-custom-code/blob/master/classes/class-nwcc-loader.php